client/notices/global-notices.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import debugModule from 'debug';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import Notice from 'components/notice';
+const debug = debugModule( 'calypso:global-notices' );
+
+var NoticesList =  React.createClass( {
```

Comment:
Should be consistent with ES6 variable declaration types, i.e. `const` here instead of `var`.

Also, an extra space after the `=` should be removed.